from utils.error.crash_it import panic
from utils.error.log_error import exception_handler

__all__ = [
    'exception_handler'
    'panic'
]